vochain/types.go

```diff
@@ -186,3 +185,11 @@ type TendermintPubKey struct {
 	Type  string `json:"type"`
 	Value []byte `json:"value"`
 }
+
+type AppStateValidators struct {
+	Address  types.HexBytes `json:"signer_address"`
+	PubKey   types.HexBytes `json:"consensus_pub_key"`
+	Power    uint64         `json:"power"`
+	Name     string         `json:"name"`
+	KeyIndex uint8          `json:"key_index"`
```

Comment:
Must `KeyIndex` be incremental? Can an existing validator change its `KeyIndex`?

Reply:
No, it cannot be changed for now.